src/patternfly/components/Check/styles.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../patternfly-utilities";
+
+.pf-c-check {
+  --pf-c-check--MarginRight: var(--pf-global--spacer--md);
+  margin-right: var(--pf-c-check--MarginRight);
```

Comment:
Should we add margin-right to the checkbox outside inline checkbox + label? When used alone, a checkbox would adjust its parent element's width.

Reply:
I think this is a good question. Currently the check component doesn't really do much other than add that margin. I believe that the design of these will change and then we can make decisions about them.